css/common.styl

```diff
@@ -166,6 +166,8 @@ $owned-card
   overflow: hidden
   text-align: left
   text-decoration: none
+  height: 270px
+  position: relative
 
   svg
```

Comment:
The project card has its own height for the svg element defined in its style file, so this is only applying to the collection card. Let's move this into the collection card style file.